.github/workflows/trigger_prerelease.yml

```
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 10 * * 3"
+
+jobs:
+  prerelease:
+    uses: newrelic/newrelic-coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
```

In case this [PR](https://github.com/newrelic/coreint-automation/pull/1) get's merged before this.
```suggestion
    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
```